doc/video.md

```diff
@@ -79,10 +96,19 @@ interface IOnScreenParticipantsUpdate {
   ]
 }
 ```
+Value of event:
+* `update: IOnScreenParticipantsUpdate` – the new participant on the screen
 
-`session.on('speakersUpdate', (ISpeakersUpdate) => {}))` - This event tells who is making noise in the conference.
 
-``` typescript
+#### `speakersUpdate`
+
+This event tells who is making noise in the conference.
+
```

Comment:
caveat: Since we can currently only update this when the on-screen user changes, this will often appear to be out of sync. This will be fixed in the future.